module/User/src/User/Service/ApiApp.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+
+namespace User\Service;
+
+use Application\Service\AbstractService;
+use Firebase\JWT\JWT;
+use User\Mapper\ApiApp as ApiAppMapper;
+use User\Model\User as UserModel;
+
+class ApiApp extends AbstractService
+{
+
+    /**
+     * @var ApiAppMapper
+     */
+    protected $mapper;
+
+    /**
+     * Constructor
+     *
+     * @param ApiAppMapper $mapper
+     */
+    public function __construct(ApiAppMapper $mapper)
+    {
+        $this->mapper = $mapper;
+    }
+
+    /**
+     * Get a callback from an appId and a user identity
+     * @param string $appId
+     * @param UserModel $user
+     * @return string
+     */
+    public function callbackWithToken($appId, UserModel $user)
+    {
+        $app = $this->getMapper()->findByAppId($appId);
+
+        $token = [
+            'iss' => 'https://gewis.nl/',
```

Comment:
It's arbitrary. According to the [spec](https://self-issued.info/docs/draft-ietf-oauth-json-web-token.html#issDef) it can be anything you want because it's up to an application to deal with it. So hardcoding gewis.nl isn't necessarily incorrect but making it dynamic might help other application distinguish if they're receiving a token from a developer or from a real user.